frontend/dockerfile/docs/rules/expose-invalid-format.md

```diff
@@ -0,0 +1,55 @@
+---
+title: ExposeInvalidFormat
+description: >-
+  IP address and host-port mapping should not be used in EXPOSE instruction. This will become an error in a future release
+aliases:
+  - /go/dockerfile/rule/expose-invalid-format/
+---
+
+## Output
+
+```text
+EXPOSE instruction should not define an IP address or host-port mapping, found '127.0.0.1:80:80'
+```
+
+## Description
+
+The [`EXPOSE`](https://docs.docker.com/reference/dockerfile/#expose) instruction
+in a Dockerfile is used to indicate which ports the container listens on at
+runtime. It should not include an IP address or host-port mapping, as this is
+not the intended use of the `EXPOSE` instruction. Instead, it should only
+specify the port number and optionally the protocol (TCP or UDP).
```

Comment:
Not sure if we do already, but we should document somewhere that `tcp` is the default if omitted (also in the OCI spec); https://github.com/opencontainers/image-spec/blob/2daaaaf0e7c16a6a147be91fde277f38573be672/config.md?plain=1#L144

```markdown
Its keys can be in the format of: `port/tcp`, `port/udp`, `port` with the default protocol being `tcp` if not specified.
```

Perhaps we should also recommend users to always include a proto. @akerouanton may have ideas on that.